Kickstarter-iOS/Views/Cells/PledgeDescriptionCell.swift

```diff
@@ -0,0 +1,155 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class PledgeDescriptionCell: UITableViewCell, ValueCell {
+  // MARK: - Properties
+
+  private lazy var rootStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var containerImageView: UIView = { UIView.init() }()
```

Comment:
Is this an image view? It's returning a normal view. Also, if it's a `UIView`, can we use the designated initializer `UIView(frame: .zero)` here?

Reply:
It's a `UIView`